yt/frontends/enzo_p/io.py

```diff
@@ -162,7 +162,7 @@ def _read_obj_field(self, obj, field, fid_data):
         ftype, fname = field
         node = "/%s/field%s%s" % (obj.block_name, self._sep, fname)
         dg = h5py.h5d.open(fid, b(node))
-        rdata = np.empty(self.ds.grid_dimensions[:self.ds.dimensionality],
+        rdata = np.empty(self.ds.grid_dimensions[:self.ds.dimensionality][::-1],
```

Comment:
It might make this line a little shorter and easier to read if you bind `self.ds.grid_dimensions[:self.ds.dimensionality][::-1]` to a variable name above this line. It might also be worth adding a comment about why you need the reverse.